docs/product/issues/issue-details/index.mdx

```diff
@@ -102,6 +102,28 @@ If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll
 
 ![Session Replay](./img/issue-replay.png)
 
+## Feature Flags
+If you’ve enabled a feature flag integration in your SDK you’ll have access to the feature flag table. If any flags were captured after the Sentry SDK's initialization and before the error event occurred, the table will contain rows of flag, evaluation-result pairs.
```

Comment:
needs link to the sdk setup instructions here

Reply:
We'd have to assume a language.  E.g. link to the Python set up instructions.

Reply:
I'm wondering if the CTA and fly out can accommodate this better?